### PR TITLE
Full stack rework

### PR DESCRIPTION
**Pull request by mdfalkow**

### CODE REVIEW

File: policy-tool-frontend/public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Policy Tool</title>
+    <title>React App</title>
```
Comment (hansidm) [inline policy-tool-frontend/public/index.html:27]: Why React App instead of Policy Tool?

File: policy-tool-frontend/public/manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "Policy Tool",
-  "name": "Policy Tool",
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (hansidm) [inline policy-tool-frontend/public/manifest.json:3]: Why React App instead of Policy Tool?

**Review by hansidm — APPROVED** [review]

File: policy-tool-frontend/public/index.html (hunk 1 above)
Comment (mdfalkow) [reply policy-tool-frontend/public/index.html:27]: Left over from the restart

File: policy-tool-frontend/public/manifest.json (hunk 2 above)
Comment (mdfalkow) [reply policy-tool-frontend/public/manifest.json:3]: Left over from the restart

**Pull request merged**